lib/helium/collection.rb

```diff
@@ -115,6 +145,10 @@ def resource_path
       uri.to_s
     end
 
+    def relationship_path
+      URI.parse("#{@belongs_to.resource_path}/relationships/#{@klass.resource_name}").to_s if @belongs_to
```

Comment:
I don't _think_ this needs to be `URI.parse`d here, I believe the http client will handle that down the line